a3p-integration/proposals/z:acceptance/recorded-retired.test.js

```diff
@@ -5,6 +5,7 @@ import { evalBundles } from '@agoric/synthetic-chain';
 const SUBMISSION_DIR = 'recorded-instances-submission';
 
 test(`recorded instances in u18`, async t => {
-  await evalBundles(SUBMISSION_DIR);
+  const result = await evalBundles(SUBMISSION_DIR);
+  console.log('recorded retired instance result:', result);
```

Comment:
I forgot to mention this and it's not worth a rev: if you tested the result you wouldn't need a `t.pass`. It would also help document what's expected